scripts/ci.sh

```diff
@@ -49,7 +49,7 @@ elif [ "$QUORUM" = true ]; then
   sudo mv docker-compose /usr/local/bin
   git clone https://github.com/jpmorganchase/quorum-examples
   cd quorum-examples
-  QUORUM_GETH_ARGS="-allow-insecure-unlock" docker-compose up -d
+  docker-compose up -d
```

Comment:
how is this change related?

Reply:
I'm happy with the rest of the code, so I defer to @gnidan's review. Seems like this change was added because Quorum changed somethings in `quorum-examples`, causing tests to fail (though I see that they're passing). I'd rather have this fixed in another PR and have this one rebased after the merge of that PR

Reply:
I agree with @seesemichaelj here. Better to make separate PRs out of unrelated changes